spec/spec_helper.rb

```diff
@@ -3,6 +3,7 @@
 require 'bundler/setup'
 require 'simplecov'
 require 'coveralls'
+require 'pry'
```

Comment:
@tpendragon Did you mean to leave `pry` in here?

Reply:
Yeah, it seemed nice. I can remove it if you want.

Reply:
My bad. I didn't notice it was in test-only code.